vocabulary/README.md

```diff
@@ -1,15 +1,7 @@
 # Vocabulary
+# Fork & Regenerate
 
-I.T. is a new language, the words may come from English but they often mean
```

Comment:
hey @Levaii  the README.md file should have its old writing. make it as so there is no change on the file. now as you can see you have the same writing in two different places.

Reply:
Didn’t know how to do they unless I delete this pull request and make an another @gelilaa

Reply:
Just copy and paste [this](https://github.com/lab-antwerp-1/home/blob/main/vocabulary/README.md) and push again